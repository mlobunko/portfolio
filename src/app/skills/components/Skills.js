import React from "react";

import Skill from "./Skill";
import { StyledSkills } from "./styles";
import ImageCss from "../images/css.svg";
import ImageGit from "../images/git.png";
import ImageHtml from "../images/html.png";
import ImageJs from "../images/js.svg";
import ImageReact from "../images/react.svg";
import ImageReduxSaga from "../images/redux-saga.png";
import ImageRedux from "../images/redux.svg";
import ImageResponsive from "../images/responsive.svg";
import ImageRestApi from "../images/rest-api.png";
import ImageSass from "../images/sass.png";
import ImageStyledComponents from "../images/styled-components.png";
import ImageTest from "../images/test.png";

export const Skills = () => (
  <StyledSkills>
    <Skill text={"HTML"} image={ImageHtml} />
    <Skill text={"CSS"} image={ImageCss} />
    <Skill text={"Sass"} image={ImageSass} />
    <Skill text={"Responsive Design"} image={ImageResponsive} />
    <Skill text={"Javascript"} image={ImageJs} />
    <Skill text={"React"} image={ImageReact} />
    <Skill text={"Redux"} image={ImageRedux} />
    <Skill text={"Redux-Saga"} image={ImageReduxSaga} />
    <Skill text={"Styled Components"} image={ImageStyledComponents} />
    <Skill text={"Git"} image={ImageGit} />
    <Skill text={"REST API"} image={ImageRestApi} />
    <Skill text={"Unit testing"} image={ImageTest} />
  </StyledSkills>
);
Skills.displayName = "Skills";

export default Skills;
