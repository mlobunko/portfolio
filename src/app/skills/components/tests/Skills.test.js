import React from "react";
import { shallow } from "enzyme";

import { Skills } from "../Skills";

describe("render", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Skills />);
    expect(wrapper).toMatchSnapshot();
  });
});
