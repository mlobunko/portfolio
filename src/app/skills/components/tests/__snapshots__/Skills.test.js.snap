// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<Skills>
  <Skill
    image="html.png"
    text="HTML"
  />
  <Skill
    image="css.svg"
    text="CSS"
  />
  <Skill
    image="sass.png"
    text="Sass"
  />
  <Skill
    image="responsive.svg"
    text="Responsive Design"
  />
  <Skill
    image="js.svg"
    text="Javascript"
  />
  <Skill
    image="react.svg"
    text="React"
  />
  <Skill
    image="redux.svg"
    text="Redux"
  />
  <Skill
    image="redux-saga.png"
    text="Redux-Saga"
  />
  <Skill
    image="styled-components.png"
    text="Styled Components"
  />
  <Skill
    image="git.png"
    text="Git"
  />
  <Skill
    image="rest-api.png"
    text="REST API"
  />
  <Skill
    image="test.png"
    text="Unit testing"
  />
</Skills>
`;
